src/test/features/claim/routes/defendant-reps-address.ts

```diff
@@ -15,7 +15,7 @@ import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
 
 const cookieName: string = config.get<string>('session.cookieName')
 
-describe("Claim issue: Defendant's representative address page", () => {
+describe('Claim issue: Defendant\'s representative address page', () => {
```

Comment:
It is done by reformatting using the style config we have on confluence. Reverted this now.